packages/client/src/raffles/__tests__/patchRaffleParticipation.test.ts

```diff
@@ -0,0 +1,41 @@
+import { participationId_1, raffleId } from 'tests/__fixtures__/raffles';
+import { RaffleParticipationStatus } from '../types';
+import client from '../../helpers/client';
+import fixtures from '../__fixtures__/patchRaffleParticipation.texture';
+import mswServer from '../../../tests/mswServer';
+import patchRaffleParticipation from '../patchRaffleParticipation';
+
+describe('patchRaffleParticipation', () => {
+  const data = {
+    value: RaffleParticipationStatus.CANCELLED,
+    path: '/status',
+    op: 'replace',
+  };
+  const expectedConfig = undefined;
+  const spy = jest.spyOn(client, 'patch');
+  const urlToBeCalled = `/checkout/v1/raffles/${raffleId}/participations/${participationId_1}`;
+
+  beforeEach(() => jest.clearAllMocks());
+
+  it('should handle a client request successfully', async () => {
+    mswServer.use(fixtures.success());
+
+    expect.assertions(2);
```

Comment:
You don't need this, as the "expect" below is being awaited. Same for every other test.